src/models/room.ts

```diff
@@ -1980,11 +1981,12 @@ export class Room extends EventEmitter {
     }
 
     public getReadReceiptForUserId(userId: string, ignoreSynthesized = false): IWrappedReceipt | null {
-        const [realReceipt, syntheticReceipt] = this.receipts["m.read"]?.[userId] ?? [];
-        if (ignoreSynthesized || realReceipt) {
+        const [realReceipt, fakeReceipt] = this.receipts["m.read"]?.[userId] ?? [];
+        if (ignoreSynthesized) {
```

Comment:
it's a little bit awkward to have the thing called different names in the function body - can we restore `syntheticReceipt` or rename `ignoreSynthesized` please?

Reply:
this sort of dual naming appears to apply throughout